tools/bin/ci_credentials.sh

```diff
@@ -47,3 +47,4 @@ write_standard_creds source-slack-singer "$SLACK_TEST_CREDS"
 write_standard_creds source-stripe-singer "$STRIPE_INTEGRATION_TEST_CREDS"
 write_standard_creds source-twilio-singer "$TWILIO_TEST_CREDS"
 write_standard_creds source-zendesk-singer "$ZENDESK_SECRETS_CREDS"
+write_standard_creds source-jira "$JIRA_INTEGRATION_TEST_CRED"
```

Comment:
nitpick just for consistency
```suggestion
write_standard_creds source-jira "$JIRA_INTEGRATION_TEST_CREDS"
```

Also, could you put this line in the right alphabetical order in the list?